### PR TITLE
Adding meta tags and images

### PR DESCRIPTION
**Pull request by AdaSeteye**
-added favicon and meta tag images
-modified the title
-added some more meta tags

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+    <meta name="twitter:image" content="/jovian_meta.png">
     <meta
       name="description"
       content="At Jovian,we're on a mission to build the world's most highly
     reputed techsnical university, and we're building it
     completely online."
     />
     <meta name="author" content="Ada" />
-    <title>My third web page</title>
+    <title>Jovian Careers</title>
```
Comment (YamBro) [inline src/index.html]: Make the title a little longer

Comment (YamBro) [inline src/index.html]: Make the title a little longer

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,28 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Learn about job openings at Jovian and submit your application now">
```
Comment (YamBro) [inline src/index.html:6]: you can make the description a little d3scriptive

File: src/index.html (hunk 1 above)
Comment (YamBro) [inline src/index.html]: The title is too short

Comment (AdaSeteye) [reply src/index.html]: Okay I will do that

File: src/index.html (hunk 2 above)
Comment (AdaSeteye) [reply src/index.html:6]: I think that is enough

File: src/index.html (hunk 1 above)
Comment (AdaSeteye) [reply src/index.html]: Done!

**Review by YamBro — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (AdaSeteye) [reply src/index.html]: Okay

**Pull request merged**